package.json

```diff
@@ -20,7 +20,7 @@
     "clean-dist": "rimraf dist",
     "build-dist-min": "webpack --bail --config webpack.config.js",
     "build-dist-dev": "webpack --bail --config webpack.config.dev.js",
-    "build-dist": "npm run clean-dist && npm run build-dist-min && npm run build-dist-dev",
+    "build-dist": "npm run clean-dist && npm run build-dist-min",
```

Comment:
Hmm, I realize now that we _do_ want to publish non-minified source files to NPMCDN. Fixing.